controllers/class-shoutem-controller.php

```diff
@@ -25,85 +25,83 @@ public function __construct( $shoutem_api, $request, $response, $dao_factory, $a
 		$this->dao_factory = $dao_factory;
 		$this->authentication = $authentication;
 		$this->caching = $caching;
-		$this->view = new ShoutemControllerView($request, $response);		
+		$this->view = new ShoutemControllerView( $request, $response );
 	}
 
 	public function ShoutemController( $shoutem_api, $request, $response, $dao_factory, $authentication, $caching ) {
 		self::__construct( $shoutem_api, $request, $response, $dao_factory, $authentication, $caching );
 	}
 
 	protected function default_paging_params() {
-		return array (
+		return array(
 			'offset' => 0,
 			'limit' => 100,
-			'version' => 1
+			'version' => 1,
 		);
 	}
-	
+
 	public function accept_standard_params_and() {
-		$params = array('session_id','offset','limit','method'); //standard params		
-		foreach(func_get_args() as $accepted_param) {	
-			$params []= $accepted_param;
+		$params = array( 'session_id','offset','limit','method' ); // standard params
+		foreach ( func_get_args() as $accepted_param ) {
+			$params [] = $accepted_param;
 		}
-		return $this->request->filter_params($params);		
+		return $this->request->filter_params( $params );
 	}
-	
-	// validating stuff 
-	
+
+	// validating stuff
 	public function validate_required_plugins() {
-		foreach(func_get_args() as $required_plugin) {	
-			if (!$this->is_plugin_active($required_plugin)) {
-				$this->response->send_error(400, "Missing required plugin");
+		foreach ( func_get_args() as $required_plugin ) {
+			if ( ! $this->is_plugin_active( $required_plugin ) ) {
+				$this->response->send_error( 400, 'Missing required plugin' );
 			}
 		}
 	}
-	
+
 	public function validate_required_params() {
-		foreach(func_get_args() as $required_param) {	
-			if (!isset($this->request->params[$required_param])) {
-				$this->response->send_error(400, "Missing required parameter $required_param");
+		foreach ( func_get_args() as $required_param ) {
+			if ( ! isset( $this->request->params[ $required_param ] ) ) {
+				$this->response->send_error( 400, "Missing required parameter $required_param" );
 			}
 		}
 	}
-			
+
 	public function validate_request_credentials() {
-		if(isset($_REQUEST['session_id'])) {
-            $session_id = trim($_REQUEST['session_id']);
-            // check if we have a Base64 string
-            if (!preg_match('{^(?:[a-z0-9+/]{4})*(?:[a-z0-9+/]{2}==|[a-z0-9+/]{3}=)?$}i', $session_id)) {
-                $session_id = '';
-            }
-			$credentials = $this->authentication->get_credentials($session_id);
-		
-			if($credentials == false || $credentials->is_valid() == false) {
-				$this->response->send_error(401, "Invalid user credentials");
+		if ( isset( $_REQUEST['session_id'] ) ) {
```

Comment:
@kasparsd If I'm looking at this correctly it is not _our_ session but rather a Shoutem API session ID that is being checked here and therefore should not violate VIP's guideliness.